IncludeToolbox/Commands/TryAndErrorRemoval_Project.cs

```diff
@@ -112,19 +103,39 @@ private bool ProcessNextFile()
                     continue;
 
                 impl.PerformTryAndErrorRemoval(document, settings);
-                ++fileIdx;
                 return true;
             }
-
-            projectItems = null;
             return false;
         }
 
+        static void RecursiveFindFilesInProject(ProjectItems items, ref Queue<ProjectItem> projectFiles)
+        {
+            var e = items.GetEnumerator();
+            while (e.MoveNext())
+            {
+                var item = e.Current;
+                if (item == null)
+                    continue;
+                var projectItem = item as ProjectItem;
+                if (projectItem == null)
+                    continue;
+                if (projectItem.FileCount > 1)
+                    RecursiveFindFilesInProject(projectItem.ProjectItems, ref projectFiles);
+                else if (projectItem.FileCount == 1)
+                {
+                    projectFiles.Enqueue(projectItem);
```

Comment:
Would be great if we could filter for C++ files here right away, but I guess I would need to change the VC++ interface for that :/